src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
1. If we are going to use this approach, then this method could be non static. 
2. method getRandomColor()  returns a random instance of the [Colors](https://github.com/TarasDovganyuk/lottery-test-task-review/pull/1/files#:~:text=public%20enum-,Colors,-%7B)  class. This method can me moved to Colors enum, as this is the logic belongs to creating instance of Colors class, and a class has to know how to create its instances.